src/Persistence.Uno.WinUI/ApplicationDataContainerSecureSettingsStorage.cs

```diff
@@ -0,0 +1,164 @@
+﻿#if WINUI
```

Comment:
Could some code be shared instead of being duplicated?

Reply:
moved duplicate code into shared